src/server/server.cc

```diff
@@ -783,6 +783,7 @@ void Server::GetServerInfo(std::string *info) {
   time(&now);
   string_stream << "# Server\r\n";
   string_stream << "version:" << VERSION << "\r\n";
```

Comment:
```suggestion
  string_stream << "kvrocks_version:" << VERSION << "\r\n";
```
I am wondering if it will be better since client frameworks can determine the server is kvrocks.

Reply:
@git-hulk how about you

Reply:
This may break the old behavior like Kvrock exporter. I prefer to keep the `version`.